.gitignore

```diff
@@ -4,6 +4,7 @@ bazel-bin
 bazel-genfiles
 bazel-out
 bazel-old
+bazel-code
```

Comment:
what's this?

Reply:
This is one of the linked executables that appeared during the build from bazel - it's on my host because I tested binding the PWD to /code (and it generated). Here is the (now dead) symlink in root's home (in the container):

```
 ls -l bazel-code 
lrwxrwxrwx 1 root root 81 Jul  5 12:04 bazel-code -> /root/.cache/bazel/_bazel_root/933c2f549343e099e3a6911b282cb71c/execroot/__main__
```
here are the whole set:

```bash
lrwxrwxrwx 1 root root 108 Jul  5 12:04 bazel-bin -> /root/.cache/bazel/_bazel_root/933c2f549343e099e3a6911b282cb71c/execroot/__main__/bazel-out/k8-fastbuild/bin
lrwxrwxrwx 1 root root  81 Jul  5 12:04 bazel-code -> /root/.cache/bazel/_bazel_root/933c2f549343e099e3a6911b282cb71c/execroot/__main__
lrwxrwxrwx 1 root root  91 Jul  5 12:04 bazel-out -> /root/.cache/bazel/_bazel_root/933c2f549343e099e3a6911b282cb71c/execroot/__main__/bazel-out
lrwxrwxrwx 1 root root 113 Jul  5 12:04 bazel-testlogs -> /root/.cache/bazel/_bazel_root/933c2f549343e099e3a6911b282cb71c/execroot/__main__/bazel-out/k8-fastbuild/testlogs
```
Was this possibly added in a newer release? If you build the container and bind to /code and then build you should see the same pop up!